discovery/src/test/java/io/airlift/discovery/client/TestServiceTypes.java

```diff
@@ -1,69 +0,0 @@
-/*
- * Copyright 2010 Proofpoint, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.airlift.discovery.client;
-
-import org.testng.annotations.Test;
-
-import static io.airlift.testing.EquivalenceTester.equivalenceTester;
-import static org.testng.Assert.assertEquals;
-
-public class TestServiceTypes
```

Comment:
The test was passing (https://travis-ci.org/airlift/airlift/builds/600146887).
However, technically, there is no need to test a library.